Woof/WoofTests/ViewModel/DetailPetSitterViewModel/DetailPetSitterViewModel+CopyToClipboardTextTests.swift

```diff
@@ -0,0 +1,59 @@
+import XCTest
+
+// swiftlint:disable:next type_name
```

Comment:
Could you please explain why do you disable this rule here?

Reply:
Fixed

Reply:
@dersim-davaod 
Because the name was longer than we allowed in the rules. But we previously discussed that this rule might be disabled for names in tests.

Reply:
The fact the rule is triggered helps you to understand that it's quite hard to read such name.

For sure, there might be cases where you can make an exception and disable rule, but first you should think "hm, probably a linter tries to help me". Given the current case, I agree with a linter.

Reply:
Yes, I agree.
The exceptions suit more for test cases, not for the name of the class.